src/main/java/org/example/algorithms/PrimsAlgorithm.java

```diff
@@ -0,0 +1,122 @@
+/* Author : G Pala Venki Reddy */
+

```

Comment:
Hi @PalaVenkiReddy, in Java, file name should be the name of public class, otherwise it doesnot compile.

Reply:
Modified and created a new pull request.